test/export/test_export.py

```diff
@@ -4504,9 +4502,22 @@ def forward(self, x):
         self.assertEqual(len(ep.state_dict), 0)
         self.assertEqual(len(ep.constants), 1)
 
-        self.assertExpectedInline(
-            str(ep.graph).strip(),
-            """\
+        if is_training_ir_test(self._testMethodName):
+            self.assertExpectedInline(
+                str(ep.graph).strip(),
+                """\
+graph():
+    %c_lifted_tensor_0 : [num_users=1] = placeholder[target=c_lifted_tensor_0]
+    %x : [num_users=2] = placeholder[target=x]
+    %lift_fresh_copy : [num_users=1] = call_function[target=torch.ops.aten.lift_fresh_copy.default](args = (%c_lifted_tensor_0,), kwargs = {})
```

Comment:
side question: the delta in these two graphs (for training vs inference) feels more like an artifact of our tracing infra than something that we tried to do. Should we spend more energy trying to clean up the extra detach() calls that show up in inference IR? (either via trying to avoid them during the tracing process, or in a graph pass)